### PR TITLE
Rook v1.10.13

### PR DESCRIPTION
**Pull request by sbruzzese902**
This PR adds the following:

- Rook v1.10.13 that supports OSD resize
- Drone pipeline with policeman only
- Support to Kubernetes v1.25
- License

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
```
Comment (ralgozino) [inline .drone.yml:1]: nit pick:
```suggestion
# Copyright (c) 2024-present SIGHUP s.r.l All rights reserved.
```

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+      note: release-notes.md
+      checksum:
+        - md5
+        - sha256
+    when:
+      ref:
+        exclude:
+          - refs/tags/v**-rc**
+        include:
+          - refs/tags/v**
```
Comment (ralgozino) [inline .drone.yml:148]: nit pick: missing NewLine at the EoF

**Review by ralgozino — CHANGES_REQUESTED** [review]
even though we are bumping just a patch version of one of the components we are also *adding compatibility to a new Kubernetes version*, let's do a v0.2.0 release of the module instead of v0.1.1. What do you think?

**Comment by sbruzzese902** [comment]
Makes sense 😄

**Review by ralgozino — APPROVED** [review]
lgtm 🚀

**Pull request merged**